Java/JDI/jdi-uitest-web/src/main/java/com/epam/jdi/uitests/web/selenium/elements/pageobjects/annotations/objects/JCheckList.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.jdi.uitests.web.selenium.elements.pageobjects.annotations.objects;
+
+import com.epam.jdi.uitests.web.selenium.elements.pageobjects.annotations.JFindBy;
+import org.openqa.selenium.support.FindBy;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.FIELD})
+public @interface JCheckList {
+    String separator() default ", ";
+    FindBy allLabels() default @FindBy();
```

Comment:
I suppose that we need to rename allLabels() to list(). In Dropdown example Roman use list().